SPECS/pcsc-lite/pcsc-lite.spec

```diff
@@ -0,0 +1,577 @@
+%global __python3	%{_bindir}/python3
+Summary:        PC/SC Lite smart card framework and applications
+Name:           pcsc-lite
+Version:        1.9.0
+Release:        2%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://pcsclite.apdu.fr/
+Source0:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2
+Source1:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2.asc
+Source2:        gpgkey-F5E11B9FFE911146F41D953D78A1B4DFE8F9C57E.gpg
```

Comment:
Similar comment for `Source1` and `Source2` here - we don't need these, should be removed. Especially since updating them for each version change would be a headache.